content/sensu-enterprise/2.6/guides/using-contact-routing.md

```diff
@@ -0,0 +1,623 @@
+{
+  "checks": {
+    "example_check": {
+      "command": "do_something.rb",
+      "handlers": [
+        "email",
+        "slack"
+      ],
+      "contacts": [
+        "support",
+        "dev"
+      ]
+    }
+  },
+  "contacts": {
+    "support": {
+      "pagerduty": {
+        "service_key": "supportfoobarkey"
+      }
+    },
+    "dev": {
+      "pagerduty": {
+        "service_key": "devfoobarkey"
+      }
+    }
+  }
+}
+{{< /highlight >}}
+
+Since both contacts match and but both contacts do not have configuration overrides for both handlers, two handler events are generated. The default configuration for both handlers will be used
+
+![Multiple Handlers with Multiple Non-Matching Contact](/images/contact-routing/multiple-handlers/multiple-non-matching-contacts.png)
+
+### Multiple Handlers with Some Matching Contacts
+
+In this example we have two handlers configured for our check, **email** and **slack**. We have three contacts being used, **support**, **dev** and **eng**. Support has a contact override for **email** and **dev** has a contact override for **slack**. **eng** has a contact override for **pagerduty** which is not a handler that the check is using.
+
+{{< highlight json >}}
+{
+  "checks": {
+    "example_check": {
+      "command": "do_something.rb",
+      "handlers": [
+        "email",
+        "slack"
+      ],
+      "contacts": [
+        "support",
+        "dev",
+        "eng"
+      ]
+    }
+  },
+  "contacts": {
+    "support": {
+      "email": {
+        "to": "support@sensuapp.io"
+      }
+    },
+    "dev": {
+      "slack": {
+        "channel": "#dev"
+      }
+    },
+    "eng": {
+      "pagerduty": {
+        "service_key": "foobarkey"
+      }
+    }
+  }
+}
+{{< /highlight >}}
+
+With some contacts having at least one single matching contact override we can expect that there will be four different handler events generated.
+
+For **email** handler we will be sending an email to the **support** contact override of "support@sensuapp.io". Since **dev** and **eng** do not have a contact override for **email** a single email will be sent to our default email configured, which in our case is "default@example.com".
+
+For **slack** handler, similar to our email handler, we'll be creating a message for our **dev** contact override to "#dev" _channel_. With **support** and **eng** both not having a contact override for **slack**, an message will be sent to our default **slack** _channel_ configured, in our case "#alerts".
+
+
+![Multiple Handlers with Some Matching Contact](/images/contact-routing/multiple-handlers/some-matching-contacts.png)
+
+# Wrapping Up
+
+## Contact Routing and Sensu Event Pipeline
+
+When a check is executed and an event is generated, Sensu splits handling
+of the event for each handler defined. When multiple handlers are defined, each
+handler is managed independently. This means that each handler is processed
+independent of any other handler configured.
```

Comment:
I feel like this could be chopped to two sentences, but also possible I am missing an intention. Just feels like the first sentence already says what the second one does. Happy chat about it!